src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -144,7 +144,8 @@ public boolean clickEditInstructorLink(int instrNum) {
     }
     
     public WebElement displayedToStudentCheckBox(int instrNum) {
-        return browser.driver.findElement(By.cssSelector("#instructorTable" + instrNum + " > div:nth-child(4) > label:nth-child(1) > input:nth-child(1)"));
+        return browser.driver.findElement(By.cssSelector("#instructorTable" + instrNum
+                                                         + " input[name=\"instructorisdisplayed\"]"));
```

Comment:
Looking through other parts of the same file, it would appear that the use of double quotes within the selector is not necessary, as single quotes would work fine as well. Extending this to cover other selectors within this file.